.github/workflows/e2e.yaml

```diff
@@ -51,8 +51,8 @@ jobs:
       env:
         TEST_KONG_CONTROLLER_IMAGE_OVERRIDE: "kong/nightly-ingress-controller:nightly"
         KONG_LICENSE_DATA: ${{ steps.license.outputs.license }}
-        KONG_CLUSTER_VERSION: ${{ matrix.kubernetes_version }}
-        ISTIO_VERSION: ${{ matrix.istio_version }}
+        KONG_CLUSTER_VERSION: ${{ matrix.kubernetes-version }}
+        ISTIO_VERSION: ${{ matrix.istio-version }}
```

Comment:
How did we get so far with this not working 🤯

Reply:
we were actually running e2e tests over the latest k8s and istio version for 16 times :rofl: